linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -428,82 +430,107 @@ if (env.PerformanceAzure == "yes") {
+        return
+    }
+    kernels_info = kernels_info.split("\n")
+    def latest_kernel_version = ""
+    kernels_info.each {
+        if (it.contains("${env.distro}${version_identifier}")) {
+            latest_kernel_version = it.split("=")[1]
         }
-        kernels_info = kernels_info.split("\n")
-        kernels_info.each {
-            if (it.contains("${env.distro}${version_identifier}")) {
-                latest_kernel_version = it.split("=")[1]
+    }
+
+    def warningMessage = ""
+
+    if (!env.KernelVersion) {
+        env.KernelVersion = latest_kernel_version
+    }
+
+    if (env.KernelVersion && latest_kernel_version && env.KernelVersion.trim() != latest_kernel_version.trim()) {
+        warningMessage = "Warning: ${env.KernelVersion} superseded by ${latest_kernel_version}!"
+    }
+    stage('Send-email') {
+        withCredentials([string(credentialsId: 'MAIL_LIST_OWNER', variable: 'MAIL_LIST_OWNER')]) {
+            def emailList = ""
+            if (env.sendMail == "yes") {
+                emailList = env.MAIL_LIST_OWNER
             }
+            emailext (
+                subject: "Please sign off ${env.KernelType} ${env.KernelVersion} kernel for ${env.distro}. ${warningMessage}",
+                to: emailList,
+                mimeType : "text/html",
+                body: '${SCRIPT, template="ubuntu.template"}',
+                recipientProviders: [[$class: 'RequesterRecipientProvider']],
+            )
         }
-        stage('Send-email') {
-            withCredentials([string(credentialsId: 'MAIL_LIST_OWNER', variable: 'MAIL_LIST_OWNER')]) {
+    }
+    stage('Kernel-Sign-Off') {
+        def owner
+        withCredentials([string(credentialsId: 'MAIL_LIST_SIGNOFF', variable: 'MAIL_LIST_SIGNOFF')]) {
+            def emailList = ""
+            if (env.sendMail == "yes") {
+                emailList = env.MAIL_LIST_OWNER
+            }
+            try {
+                checkout scm
+                echo "Ask for manual sign-off from the owner"
+                timeout(time: 72, unit: 'HOURS') {
+                    signoff = input(message: "Are you sure you want to sign off ${env.distro} ${env.KernelType} ${env.KernelVersion} kernel? ${warningMessage}",
+                        parameters: [string(defaultValue: 'Press Proceed/Abort after verifying the test results', description: '', name: '')],
+                        submitterParameter: "USER")
+                    owner = signoff["USER"].toString()
+                }
+                env.SIGNED_OFF_BY = owner
+                env.SIGNED_OFF_RESULT = "true"
                 emailext (
-                    subject: "Please sign off ${latest_kernel_version} ${env.kernel} kernel for ${env.distro}",
-                    to: "${env.MAIL_LIST_OWNER}",
+                    subject: "${env.KernelType} ${env.KernelVersion} kernel for ${env.distro} was signed off by ${env.SIGNED_OFF_BY}",
```

Comment:
on the abort case the the subject variables are `subject: "${env.KernelVersion} ${env.KernelType} kernel`

Reply:
done